test/tests/actions/tests

```diff
@@ -0,0 +1,18 @@
+[Tests]
+  issues = '#248'
+  design = 'EqualValueEmbeddedConstraintAction.md'
+
+  [EqualValueEmbeddedConstraintAction]
+    requirement = 'This action should apply constraint for all specified '
```

Comment:
The system shall set up a set of equal value constraints tying the nodes in multiple blocks to a primary block for multiple variables.

Reply:
Corrected!